### ISSUE TITLE
Geo Map should be part of tab view as well as full-screen

### ISSUE THREAD

**Issue by MarshallBrinn** [GENI-NSF/geni-portal#1264]
The new Geo Map view for slice resources should be part of the slice-jacks page on a separate "Geographic View" tab. And from this view one can click on an "expand" button to get a new window with a map that takes up the full browser space. 

We should make sure that the tab geo-view doesn't request the manifests until it is opened (much as the Jacks view does).

 _Imported from trac ticket [#1264](http://trac.gpolab.bbn.com/proto-ch/ticket/1264),  created by mbrinn on 01-13-2015 at 09:07, last modified: 01-16-2015 at 12:30_

**Comment by ahelsing** [comment]
2 nice to haves:

The full window map should have the standard footer if possible.

The tab view should share manifests with the Jacks view if possible.

_Trac comment by ahelsing on 01-13-2015 at 09:19_

**Comment by MarshallBrinn** [comment]
The main goal of this ticket is fixed: there is an internal tab for the geo map as well as an expanded view. Plus we added the footer.

Two things weren't done.

1. I didn't ensure that the geo-view doesn't request manifest until it is opened. (The jacks view doesn't do this either).

2. I didn't make the jacks view and the map view share manifests. Turns out this is very hard since one needs to do this on the server side and one needs to do things on the client side. Or at least the way things are architected. I'll open up a separate ticket to track this one and close this one.

_Trac comment by mbrinn (github user: MarshallBrinn) on 01-16-2015 at 12:27_
